_security/authentication-backends/ldap.md

```diff
@@ -155,7 +155,7 @@ By default, the Security plugin validates the TLS certificate of the LDAP server
 
 ```
 plugins.security.ssl.transport.pemtrustedcas_filepath: ...
```

Comment:
I need to double check the logic in the security plugin. These settings are used if `pemtrustedcas_filepath` or `pemtrustedcas_content` are not explicitly set on the LDAP Authz backend config. 

Relevant code: https://github.com/opensearch-project/security/blob/main/src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthorizationBackend.java#L585-L607

From that block of code it looks the following settings would need to be configured (not necessarily all of them):

```
plugins.security.ssl.transport.truststore_filepath: ...
plugins.security.ssl.transport.truststore_password: ...
plugins.security.ssl.transport.truststore_type: ...
plugins.security.ssl.transport.keystore_filepath: ...
plugins.security.ssl.transport.keystore_password: ...
plugins.security.ssl.transport.keystore_type: ...
```

It looks like this is the fallback config if LDAP is not explicitly configured with TLS settings. i.e. it tries to connect to LDAP with the trusted CAs that the transport layer is configured with.